config/config_test.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_LoadFromPath(t *testing.T) {
```

Comment:
Since this comment and the one about `warned` depends on another, can you give me your opinion here: https://github.com/rancher/cli/pull/341#discussion_r1330444926?

Reply:
Even if you deliberately ran the tests sequentially, that global state with `warned` is not really needed and thus best to avoid.

Reply:
I understand that it's best to be avoided, but the reason I put `warned` is explained in the comment I linked above. Basically, I have chosen to warn only once during the invocation of the CLI instead of everytime the config is read. This is what `warned` does. In light of that, do you have a suggestion? (I'd suggest replying in the comment linked so it's easier to follow)

Reply:
All tests are running in parallel now.